hw2/spec/lib/router_spec.rb

```diff
@@ -24,6 +24,23 @@
     end
   end
 
+  context 'when request is GET' do
+    let(:env) { { 'REQUEST_PATH' => '/post/1', 'REQUEST_METHOD' => 'GET'} }
+
+    it 'matches request' do
+      expect(subject.call(env)).to eq [200, {}, ['post show page']]
+    end
+  end
+
+
+   context 'when request is GET' do
```

Comment:
contexts should not have the same text
on line 27 name could be `when request has params in path`
and second one `when request has unknown path`